net/net.sh

```diff
@@ -840,6 +840,32 @@ stop() {
   echo "Stopping nodes took $SECONDS seconds"
 }
 
+
+checkPremptibleInstances() {
```

Comment:
Would it make sense to drop a `preemptible` flag in the config so we can gate this logic?

Reply:
We don't actually have a way in net.sh to tell if we have any preemptible instances in the cluster, so I opted for all the time instead of extending the config file to include that bit.  Also I figured, well, if you can't ping the dedicated instances then you're probably in a bad place too.